### PR TITLE
Kinflate add generic secret command.

### PR DESCRIPTION
**Pull request by seans3**

### CODE REVIEW

File: pkg/kinflate/commands/secret.go [hunk 1]
```diff
@@ -63,6 +78,35 @@ func newCmdAddSecretGeneric(errOut io.Writer) *cobra.Command {
...
+	return nil
+}
+
+func getOrCreateGenericSecret(m *manifest.Manifest, name string) *manifest.GenericSecret {
+	for i, s := range m.GenericSecrets {
+		if name == s.Name {
+			return &m.GenericSecrets[i]
+		}
+	}
+	// config map not found, create new one and add it to the manifest.
```
Comment (apelisse) [inline pkg/kinflate/commands/secret.go]: update comment

File: pkg/kinflate/commands/secret.go [hunk 2]
```diff
@@ -63,6 +78,35 @@ func newCmdAddSecretGeneric(errOut io.Writer) *cobra.Command {
...
+	gs := &manifest.GenericSecret{Name: name}
+	m.GenericSecrets = append(m.GenericSecrets, *gs)
```
Comment (apelisse) [inline pkg/kinflate/commands/secret.go]: why do you get a pointer, if the goal is to push a copy? (that probably already applied to configmap ...)

File: pkg/kinflate/commands/secret.go [hunk 3]
```diff
@@ -45,12 +51,21 @@ func newCmdAddSecretGeneric(errOut io.Writer) *cobra.Command {
 				return err
 			}
 
-			if len(args) != 1 {
-				return fmt.Errorf("error: exactly one NAME is required, got %d", len(args))
+			// Load in the manifest file.
```
Comment (apelisse) [inline pkg/kinflate/commands/secret.go]: Not a big fan of this kind of comments :-)

File: pkg/kinflate/commands/secret.go (hunk 1 above)
Comment (seans3) [reply pkg/kinflate/commands/secret.go]: Fixed

File: pkg/kinflate/commands/secret.go (hunk 2 above)
Comment (seans3) [reply pkg/kinflate/commands/secret.go]: Fixed

**Pull request merged**